docs/monthly-meeting/2022-03.md

```diff
@@ -0,0 +1,130 @@
+# Documentation Community's Team Meeting (March 2022)
+
+- **Date:** 2022-03-07
+- **Time:** [22:00 UTC](https://arewemeetingyet.com/UTC/2022-02-07/22:00/Docs%20WG)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **GitHub issue:** Agenda Planning Issue
+- **Calendar for future meetings:** XXX
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+- name / affiliation / `@github-username`
+- Petr / RedHat, SC / `@encukou`
+- Mariatta / Core Dev / `@mariatta`
+- Pradyun / ??? / `@pradyunsg`
+- CAM / NASA, PEP infra team / `@CAM-Gerlach`
+- James Gerity / random user / `@SnoopJeDi`
+- Julien Palard / Core Dev / `@JulienPalard`
+- Ned / Community member /
+- Adam /
+
+
+## Quick updates - Introductions
+
+- [name=Julien] https://github.com/sphinx-contrib/sphinx-lint
+  - fork of the rstlint.py from CPython, as an independent project
+
+## Reports and celebrations
+
+
+* Shout out to @AA-Turner, Barry, Mariatta and everyone who contributed for getting PEP 676 over the finish line! Its a huge improvement both for us PEP editors, and the Python community at large. — [name=CAM] 
+
+
+## Agenda items
+
+
+
+### Clean up, fix and improve grammar, phrasing, syntax/links and overall prose quality in charter
+
+- Who should review [PR29](https://github.com/python/docs-community/pull/29)? What's the official status of the charter?
+
+### Docs and PEPs
+
+- Is the Docs Community concerned with PEP editing? (Devguide?)
+  - Seems like the PEP editors are more concerned with that, since they are the editorial board and implementation team for PEPs like the Docs WG and Docs Team is for PEPs, though on broader or cross-cutting issues that affect both (e.g. what should be a PEP vs. go elsewhere, harmonizing common standards) then it would be good to involve the latter as well. — CAM
+  - Cross-cutting or broad issues -- perhaps the documentation community forum is a good one for collaboration and forming consensus.
+
+### Python Language Summit @ PyCon US
+
+- If there's any updates/issues that would be relavant to the Python core developers, this would be a good place for presenting that for discussion!
+- Still in planning stage, but worth it to start thinking about it now, if you want to bring up something for discussion.
+- [name=Petr] gave a summary of what the event will look like.
+  - In-person event, at PyCon US 2022.
+- What would the presentation be about?
+  - Updates, items that need support from core devs
+- Pay attention to `docs-community` and/or `python-dev` to fill out a form
+- We talked about the docs group at the last 2 lang summits. This time the group has started!
+- Desires for revamping the main CPython documentation theme might be a topic for discussion?
+- Lightning talk sounds good
+
+### Structuring Docs, Devguide, and PEPs
+
+
+[We didn't get to this]
```

Comment:
```suggestion
[We didn't get to this, though notes of the discussion triggered by this is below]
```

Reply:
Oh, wait, why did I put that comment here? We did get to this.
Thanks for the catch!